<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>1266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: white
}</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>911</width>
      <height>1111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget{
background-color: none;
height: 0px;
width: 0px;
padding: 0px;
margin: 0px;
border: 0px
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>0</y>
        <width>931</width>
        <height>1121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollArea{
	background-color: white;
	border:0px solid white
}</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scroll_contents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>931</width>
         <height>1121</height>
        </rect>
       </property>
       <widget class="QLabel" name="prog_lbl">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>10</y>
          <width>301</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	background-color: white;
	color: #1D1D1D;
	font: 500 35pt &quot;Epilogue&quot;;
}</string>
        </property>
        <property name="text">
         <string>Today's Progress</string>
        </property>
       </widget>
       <widget class="QLabel" name="day_lbl">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>70</y>
          <width>351</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	background-color: white;
	color: #939393;
	font: 15pt &quot;Epilogue&quot;;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="cal">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>110</y>
          <width>351</width>
          <height>211</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
background-color:white;
border: 1px solid #C0C0C0;
border-radius:10px;
font:15px &quot;Epilogue&quot;;
color:#C0C0C0;
padding-bottom:90;
padding-left:137
}</string>
        </property>
        <property name="text">
         <string>Calories</string>
        </property>
       </widget>
       <widget class="QLabel" name="prot">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>110</y>
          <width>351</width>
          <height>211</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
background-color:white;
border: 1px solid #C0C0C0;
border-radius:10px;
font:15px &quot;Epilogue&quot;;
color:#C0C0C0;
padding-bottom:90;
padding-left:142
}</string>
        </property>
        <property name="text">
         <string>Protein</string>
        </property>
       </widget>
       <widget class="QLabel" name="fat">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>350</y>
          <width>351</width>
          <height>211</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
background-color:white;
border: 1px solid #C0C0C0;
border-radius:10px;
font:15px &quot;Epilogue&quot;;
color:#C0C0C0;
padding-bottom:90;
padding-left:162
}</string>
        </property>
        <property name="text">
         <string>Fat</string>
        </property>
       </widget>
       <widget class="QLabel" name="carb">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>350</y>
          <width>351</width>
          <height>211</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
background-color:white;
border: 1px solid #C0C0C0;
border-radius:10px;
font:15px &quot;Epilogue&quot;;
color:#C0C0C0;
padding-bottom:90;
padding-left:142
}</string>
        </property>
        <property name="text">
         <string>Carbs</string>
        </property>
       </widget>
       <widget class="QLabel" name="cal_counter">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>180</y>
          <width>101</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 700 40pt &quot;Epilogue&quot;;
	color:#1D1D1D
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="prot_counter">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>180</y>
          <width>101</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 700 40pt &quot;Epilogue&quot;;
	color:#1D1D1D
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="carbs_counter">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>420</y>
          <width>101</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 700 40pt &quot;Epilogue&quot;;
	color:#1D1D1D
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="fats_counter">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>420</y>
          <width>101</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 700 40pt &quot;Epilogue&quot;;
	color:#1D1D1D
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="carb_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>580</y>
          <width>761</width>
          <height>311</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
background-color:white;
border: 1px solid #C0C0C0;
border-radius:10px;
font:30px &quot;Epilogue&quot;;
color:#1D1D1D;
padding-bottom:220;
padding-left:10
}</string>
        </property>
        <property name="text">
         <string>Today's Meals</string>
        </property>
       </widget>
       <widget class="QPushButton" name="scan_btn">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>920</y>
          <width>761</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#1D1D1D;
	font: 600 25pt &quot;Epilogue&quot;;
	border-radius:25px
}</string>
        </property>
        <property name="text">
         <string>Scan Food</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>1000</y>
          <width>761</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:white;
	font: 600 25pt &quot;Epilogue&quot;;
	border-radius:25px;
	color: #1D1D1D;
	border:1px solid #C0C0C0
}</string>
        </property>
        <property name="text">
         <string>Recipes</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>640</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	background-color:white;
	font:200 12px &quot;Epilogue&quot;;
	color:#C0C0C0
}</string>
        </property>
        <property name="text">
         <string>No logged in recipes</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>821</width>
        <height>701</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	border:1px solid #C0C0C0;
	border-radius: 10px
}</string>
      </property>
      <widget class="QLineEdit" name="search_bar">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>761</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	color:#C0C0C0;
	font: 300 13pt &quot;Epilogue&quot;;
	padding-left:5px
}
</string>
       </property>
       <property name="text">
        <string>Select Ingredients</string>
       </property>
      </widget>
      <widget class="QLabel" name="igr_select">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 300 20pt &quot;Epilogue&quot;;
	color:#1D1D1D;
	border:none;
}</string>
       </property>
       <property name="text">
        <string>Select Ingredients</string>
       </property>
      </widget>
      <widget class="QWidget" name="ingr_add" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>801</width>
         <height>331</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color: none;
font: 800 25px &quot;Epilogue&quot;;
color: #1D1D1D
}
</string>
    </property>
    <property name="text">
     <string>NutriFit</string>
    </property>
   </widget>
   <widget class="QFrame" name="dashboard">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
border: none;
border-radius: 5px
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="dashboard_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: white;
	font: 500 15pt &quot;Epilogue&quot;;
	color:#1D1D1D;
	border-radius: 10px;
	border-color: none
}
QPushButton:checked {
	background-color:	#EFEFEF;
}</string>
     </property>
     <property name="text">
      <string>Dashboard</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="recipe">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
border: none;
border-radius: 5px
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="recipe_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: white;
	font: 500 15pt &quot;Epilogue&quot;;
	color:#1D1D1D;
	border-radius: 10px;
	border-color: none
}
QPushButton:checked {
	background-color:	#EFEFEF;
}</string>
     </property>
     <property name="text">
      <string>Recipe</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="exercise">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
border: none;
border-radius: 5px
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="exercise_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: white;
	font: 500 15pt &quot;Epilogue&quot;;
	color:#1D1D1D;
	border-radius: 10px;
	border-color: none
}
QPushButton:checked {
	background-color:	#EFEFEF;
}</string>
     </property>
     <property name="text">
      <string>Exercise</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="meditation">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
border: none;
border-radius: 5px
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="mediation_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: white;
	font: 500 15pt &quot;Epilogue&quot;;
	color:#1D1D1D;
	border-radius: 10px;
	border-color: none
}
QPushButton:checked {
	background-color:	#EFEFEF;
}</string>
     </property>
     <property name="text">
      <string>Meditation</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="profile">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
border: none;
border-radius: 5px
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="profile_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: white;
	font: 500 15pt &quot;Epilogue&quot;;
	color:#1D1D1D;
	border-radius: 10px;
	border-color: none
}
QPushButton:checked {
	background-color:	#EFEFEF;
}</string>
     </property>
     <property name="text">
      <string>Profile</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>831</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line{
	height:1px;
	background-color: #e8e8e8;
	border:none;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
